docs/administration/datagrid/fields.md

```diff
@@ -0,0 +1,83 @@
+# Fields
+
+Fields are the building blocks of a datagrid. They are used to define the columns of the datagrid and to specify how the data should be displayed.
+
+## Configuration
+
+Fields are defined by the `Datagrid` class using the `add(string $name, array $options)`, `update(string $name, array $options)` and `remove(string $name)` methods. The `$name` parameter is the name of the field and the `$options` parameter is an array of options that define the behavior of the field.
+
+In case you are using `OrmAdapter`, you can use dot notation to access nested properties of the entity. Here are some examples of how to define field names:
+
+- `$datagrid->add('domainId')` - This will fetch the `domainId` property of the entity.
+- `$datagrid->add('currency.code')` - This will join the `currency` relation and fetch the `code` property of the `Currency` entity.
+- `$datagrid->add('status.name')` - This will join the `status` relation and look for the `name` property of the `Status` entity. In this case `Status` is a translatable entity and the `name` property is a translatable field. The datagrid will automatically fetch the correct translation based on the current locale.
```

Comment:
It would be nice to automatically provide a fallback text if the translation is not entered. 
Something similar to the current product grid behavior

![Snímek obrazovky 2025-03-05 v 10 26 02](https://github.com/user-attachments/assets/6c125f64-1c70-4b3f-bc4d-b35cfc6ba25b)

now the empty field is rendered

Reply:
You can do that with transform or custom template. We can add this functionality later